packages/core/src/components/Dialog/Dialog.jsx

```diff
@@ -0,0 +1,120 @@
+import AriaModal from 'react-aria-modal';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export const Dialog = function(props) {
+  const {
+    actions,
+    ariaCloseLabel,
+    children,
+    escapeExitDisabled,
+    onExit,
+    title,
+    ...modalProps
+  } = props;
+
+  return (
+    <AriaModal
+      dialogClass="ds-c-dialog"
+      escapeExits={!escapeExitDisabled}
+      includeDefaultStyles={false}
+      onExit={onExit}
+      titleId="dialog-title"
+      underlayClass="ds-c-dialog-wrap"
+      {...modalProps}
+    >
+      <div role="document">
+        <header className="ds-c-dialog__header" role="banner">
```

Comment:
> Let's add a `role="banner"` to this header element, for backward compatibility.

@1Copenut Is it better to add `role="banner"` or change this element to a normal `div`?

Reply:
Let's leave the `role="banner"` because it's what's needed for backward compatibility. NVDA and VoiceOver announce the landmarks, and VoiceOver allows users to navigate inside the modal via landmarks, so having these are a nice feature.